homeassistant/components/lyric/sensor.py

```diff
@@ -84,7 +84,9 @@ def get_setpoint_status(status: str, time: str) -> str | None:
                             name="Indoor Temperature",
                             device_class=SensorDeviceClass.TEMPERATURE,
                             state_class=SensorStateClass.MEASUREMENT,
-                            native_unit_of_measurement=hass.config.units.temperature_unit,
+                            native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT
+                            if device.units == "Fahrenheit"
+                            else UnitOfTemperature.CELSIUS,
```

Comment:
Avoid the multiline ternary; they are hard to read.

Reply:
Done!